client/main.html

```diff
@@ -1,6 +1,6 @@
 <head>
   <title>Unee-T</title>
-  <meta name="viewport" content="width=device-width, height=device-height, initial-scale=1">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=0, minimum-scale=1.0, maximum-scale=1.0">
```

Comment:
Shouldn't we keep `height=device-height`?

Reply:
I shamelessly copied from view-source:https://busrouter.sg/ which behaves as I expect mobile Web app to do in Safari. i.e. double tap zoom is disabled.

Reply:
I'm halfway sure that it had a reason for being there. If remove it we need to verify our layouts still behave as before across all platforms

Reply:
:man_shrugging: